meta.yaml

```diff
@@ -63,7 +63,7 @@ test:
     - fre.catalog
   commands:
     - pylint --max-args 6 -ry --ignored-modules netCDF4,cmor fre/ || echo "pylint returned non-zero exit code and will kill the workflow. guarding against this now."
-    - pip install GitPython && pytest --config-file=fre/pytest.ini --cov-config=fre/coveragerc --cov=fre fre/
+    - pip install GitPython && pytest --config-file=fre/pytest.ini --cov-config=fre/coveragerc --cov=fre --ignore=fre/make/tests/compilation fre/ # run pytest but ignore any tests that require compilation
```

Comment:
mmmmm maybe we need a [`pytest.ini`](https://docs.pytest.org/en/stable/reference/customize.html) ?

Reply:
To ignore the tests? Or to add the directory?

The parent directory is in the current pytest.ini, which should cover this as well I think:
https://github.com/NOAA-GFDL/fre-cli/blob/20e943aa94b04ed2e50bfa37d19920eedba13c2b/fre/pytest.ini#L8

Reply:
ah i'm wondering if we can use it to ignore, but maybe we could do an explicit allow-list too, the list wouldn't be too long... yet.

Reply:
that being said this definitely works, so what i'm pointing out is really a nit for now, priority wise

Reply:
It looks like there's a way to add flags to that file, but if the ignore flag was added it would always be skipped by pytest, not just during conda builds.